esp/src/eclwatch/css/hpcc.css

```diff
@@ -260,6 +260,10 @@ hr.dashedLine {
     white-space: normal; 
 }
 
+.justify-right {
+    text-align: right !important; 
```

Comment:
Is the `!important` needed?  In general it should only be used as a last measure (adding a more specific CSS selector may be a better approach).

Reply:
Yes, we are using the "!important" because the parent class overwrites this every time otherwise.

Reply:
You might be able to "win" with better CSS specificity?